qiskit/visualization/latex.py

```diff
@@ -51,11 +55,13 @@ def __init__(
         reverse_bits=False,
         plot_barriers=True,
         layout=None,
+        with_layout=False,
         initial_state=False,
         cregbundle=False,
         global_phase=None,
         qregs=None,
         cregs=None,
+        circuit=None,
```

Comment:
In general, it is better to add parameters at the end, in case somebody is calling the method with positional arguments. I dont think is a biggy in this case. 

```suggestion
        initial_state=False,
        cregbundle=False,
        global_phase=None,
        qregs=None,
        cregs=None,
        with_layout=False,
        circuit=None,
```